### PR TITLE
Removed commented out code from start of Rails 5 upgrade [#177681375]

### PR DESCRIPTION
**Pull request by dougmartin**

### CODE REVIEW

**Comment by knowuh** [comment]
@scytacki in 41743de we monkey patched ActiveRecord::Base and the tests started passing again.

in 0ebc6f3 we Fic'd Mr.  ;)

File: rails/config/initializers/delayed_web.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Tell Delayed::Web that we're using ActiveRecord as our backend.
+Rails.application.config.to_prepare do
+  Delayed::Web::Job.backend = 'active_record'
+end
```
Comment (scytacki) [inline rails/config/initializers/delayed_web.rb:4]: I thought this was removed?

**Review by scytacki — APPROVED** [review]
looks good to me, as far as I can tell

File: rails/config/initializers/delayed_web.rb (hunk 1 above)
Comment (dougmartin) [reply rails/config/initializers/delayed_web.rb:4]: This is for the replacement gem.  We replaced `delayed_job_web` with `delayed-web` and it requires this new configuration fie.

**Comment by dougmartin** [comment]
I had to rebase this to `rails5` as it was based on a commit that I deleted (the commit was instead moved to master).

**Pull request merged**